utils/build-script

```diff
@@ -839,8 +891,35 @@ the number of parallel build jobs to use""",
     for v in ['MAKEFLAGS', 'SDKROOT', 'MACOSX_DEPLOYMENT_TARGET', 'IPHONEOS_DEPLOYMENT_TARGET']:
         os.environ.pop(v, None)
 
+    if args.show_sdks:
+        swift_build_support.debug.print_xcodebuild_versions([
+            'iphonesimulator',
+            'appletvsimulator',
+            'watchsimulator',
+        ])
+
     check_call(build_script_impl_args)
 
+    if args.symbols_package:
+        symbols_package_path = os.path.abspath(args.symbols_package)
```

Comment:
So do you think I should remove this call to `abspath()`, then? I'd be happy to do so.